target/linux/ramips/image/mt7621.mk

```diff
@@ -1475,6 +1475,16 @@ define Device/linksys_re6500
 endef
 TARGET_DEVICES += linksys_re6500
 
+define Device/linksys_re7000
+  $(Device/uimage-lzma-loader)
+  IMAGE_SIZE := 16064k
+  DEVICE_VENDOR := Linksys
+  DEVICE_MODEL := RE7000
+  DEVICE_PACKAGES := kmod-mt7603 kmod-mt7615-firmware
+  SUPPORTED_DEVICES += re7000
```

Comment:
`SUPPORTED_DEVICES` is only needed if you want to use sysupgrade from an older OpenWrt installation where the device was named `re7000`. I think there is no such installation, then you can remove this line.